packages/cozy-harvest-lib/src/components/Routes/RoutesV6.jsx

```diff
@@ -57,49 +45,44 @@ const RoutesV6 = ({
       />
 
       {flag('harvest.inappconnectors.enabled') ? (
-        <Route
-          path="accounts/:accountId"
-          element={
-            <HarvestParamsWrapper>
-              {params => (
-                <AccountModalWithoutTabs
-                  konnector={konnectorWithTriggers}
-                  accountId={params.accountId}
-                  accountsAndTriggers={accountsAndTriggers}
-                  showNewAccountButton={!konnectorWithTriggers.clientSide}
-                  showAccountSelection={!konnectorWithTriggers.clientSide}
-                  onDismiss={onDismiss}
-                />
-              )}
-            </HarvestParamsWrapper>
-          }
-        >
+        <>
           <Route
-            index
+            path="accounts/:accountId"
             element={
-              <AccountModalContentWrapper>
-                <DataTab
-                  konnector={konnectorWithTriggers}
-                  showNewAccountButton={!konnectorWithTriggers.clientSide}
-                  onDismiss={onDismiss}
-                />
-              </AccountModalContentWrapper>
+              <HarvestParamsWrapper>
+                {params => (
+                  <AccountModal
+                    konnector={konnectorWithTriggers}
+                    accountId={params.accountId}
+                    accountsAndTriggers={accountsAndTriggers}
+                    onDismiss={onDismiss}
+                    showNewAccountButton={!konnectorWithTriggers.clientSide}
+                    showAccountSelection={!konnectorWithTriggers.clientSide}
+                    component="DataTab"
+                  />
+                )}
+              </HarvestParamsWrapper>
             }
           />
           <Route
-            path="config"
+            path="accounts/:accountId/config"
             element={
-              <AccountModalContentWrapper>
-                <ConfigurationTab
-                  konnector={konnectorWithTriggers}
-                  showNewAccountButton={!konnectorWithTriggers.clientSide}
-                  onAccountDeleted={onDismiss}
-                  addAccount={() => navigate('new', { replace: true })}
-                />
-              </AccountModalContentWrapper>
+              <HarvestParamsWrapper>
+                {params => (
+                  <AccountModal
+                    konnector={konnectorWithTriggers}
+                    accountId={params.accountId}
+                    accountsAndTriggers={accountsAndTriggers}
+                    onDismiss={onDismiss}
+                    showNewAccountButton={!konnectorWithTriggers.clientSide}
+                    showAccountSelection={!konnectorWithTriggers.clientSide}
+                    component="ConfigurationTab"
+                  />
+                )}
+              </HarvestParamsWrapper>
             }
           />
-        </Route>
+        </>
       ) : (
         <Route
           path="accounts/:accountId"
```

Comment:
@JF-Cozy / @Merkur39 Can you review this change please?

Reply:
sur la route ça me semble ok, sur les enfants rendus, après discussion avec @Merkur39 je lui laisse la main pour répondre (il a plus le sujet en tête que moi)

Reply:
Ca me parait OK, n'utilisant plus un parent commun (`AccountModalWithoutTabs`), dans ce cas présent autant simplifier